### ISSUE TITLE
empty array after parsing

### ISSUE THREAD

**Issue by tadiraman** [freesoftwarefactory/parse-multipart#20]
hey,

I am having same issue as here https://github.com/freesoftwarefactory/parse-multipart/issues/15

var parts = multipart.Parse(body, boundary);
parts is empty!?

**Comment by s11shweta** [comment]
hi,

i am facing same issue. the parts array is always empty...
please advice

thanks

**Comment by tadiraman** [comment]
Hi,

This method didn't work as expected. So we created our own parser at buffer level. Let me post a gist of it soon.

**Comment by cityofchandler** [comment]
Same here.

**Comment by tadiraman** [comment]
@s11shweta  and others: Apologies for delay. Here is the gist which handles buffer extraction which takes care of the Boundary.
https://gist.github.com/tadiraman/6cb2a1770109206bb67a003e25a6de7e

**Comment by damon-kreft** [comment]
Yeah the example from the README doesn't even work. Presumably a dead broken library now

**Comment by hanktrizz** [comment]
Same here. Always getting an empty array.

**Comment by downwater** [comment]
Confirmed here. An SO answers shows a simple and useful way to parse multipart/byteranges response: https://stackoverflow.com/a/41351254
